tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/CredentialTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.AppConfig;
+using Microsoft.Identity.Client.ManagedIdentity;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Microsoft.Identity.Test.Common.Core.Helpers.ManagedIdentityTestUtil;
+
+namespace Microsoft.Identity.Test.Unit.ManagedIdentityTests
+{
+    [TestClass]
+    public class CredentialTests : TestBase
+    {
+        [TestMethod]
+        public async Task CredentialEndpointBadRequestTestAsync()
```

Comment:
This test fails on my machine with

Exception Message: Assert.AreEqual failed. Expected:<http://169.254.169.254/metadata/identity/credential>. Actual:<http://169.254.169.254/metadata/identity/oauth2/token>.